src/view_element.cpp

```diff
@@ -718,6 +718,9 @@ void View::DrawCustos(DeviceContext *dc, LayerElement *element, Layer *layer, St
     }
 
     dc->EndGraphic(element, this);
+
+    /************ Draw children (accidentals, etc) ************/
+    DrawLayerChildren(dc, custos, layer, staff, measure);
```

Comment:
Drawing the children should be done before ending the graphic. Otherwise the SVG tree will not match the MEI one

Reply:
Thanks for the comments. I've added those changes